nixos/modules/flyingcircus/roles/loghost.nix

```diff
@@ -69,6 +69,35 @@ let
         "echo -n $text | sha256sum | cut -f1 -d \" \" > $out")
       );
 
+  logstashSSLHelper = pkgs.writeScriptBin "logstash_ssl_import" ''
+    #!${pkgs.bash}/bin/bash
+    set -e
+
+    puppet="puppet.$FCIO_LOCATION.gocept.net"
+    hostname="$(hostname)"
```

Comment:
There are at least 4 sources of hostname information: the $HOSTNAME environment variable, /etc/hostname content, the kernel hostname data structure and the ENC host name. The latter is FCIO-specific. The `hostname` command uses the first 3 sources, but I'm not sure about the priority. 

It would probably be better to use $FCIO_HOSTNAME (=ENC hostname) in this script.